packages/@o3r/localization/builders/metadata-check/helpers/localization-metadata-comparison.helper.ts

```diff
@@ -0,0 +1,43 @@
+import { JSONLocalization, LocalizationMetadata } from '@o3r/localization';
+import { MetadataComparator } from '@o3r/extractors';
+
+/**
+ * Interface describing a localization migration element
+ */
+export interface MigrationLocalizationMetadata {
+  key: string;
+}
+
+/**
+ * Comparator used to compare one version of localization metadata with another
+ */
+export class LocalizationMetadataComparator implements MetadataComparator<JSONLocalization> {
+  /**
+   * Returns an array of localization metadata from a metadata file.
+   * @param content Content of a migration metadata file
+   */
+  public getArray(content: LocalizationMetadata): JSONLocalization[] {
+    return content;
+  }
+
+  /**
+   * @inheritdoc
+   */
+  public getName(config: JSONLocalization): string {
```

Comment:
getKey ?

Reply:
It has to follow the common interface for all extractors so getName could maybe renamed to getIdentifier into MetadataComparator